.github/changelog.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+m_branch=m;
+changelog_file=CHANGELOG.md;
```

Comment:
Shellcheck says:
```
warning: changelog_file appears unused. Verify use (or export if used externally). [SC2034]
```
Not really important.

(I really recommend shellcheck btw)

Reply:
I think this is the first time I write bash in a year, so it's not worth installing it :smiley:

Reply:
Removed unused variable